### PR TITLE
Breast Cancer Classification using Deep Neural Networks (Tensorflow)

### PR DESCRIPTION
**Pull request by karthiksekaran**
This project is developed to classify breast cancer. The data set is taken from UCI Repository (Wisconsin Breast Cancer Original Data set). It has binary class labels as 2 and 4 which represents benign and malignant respectively. There are 9 features given in the data set. Out of which only 4 features are selected based on Recursive Feature Elimination Technique for classification process. Deep Neural Network was built using deep learning library called Tensorflow. This network has 4 input nodes in input layer, 3 hidden layers with 20, 10, 20 neurons in each layer respectively and finally an output layer with two neurons, one for benign and another one is for malignant classification. This network uses Rectified Linear Unit (ReLU) as an activation function. This system achieves a maximum of 99.60% of accuracy after multiple iterations.

### CODE REVIEW

**Comment by karthiksekaran** [comment]
Pre-Processing and Feature Selection Process Source Codes are not uploaded due to personal reasons.
